### DISCUSSION TITLE
Getting Error when fetching tables from googlesheets

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#5660] [Help]
Getting error when fetching tables from google sheets : "Error updating datasource schema" 
how to resolve this ?

TIA

**Comment by melohagan** [comment]
Hi @newtonlaw95 

Are you self hosting or cloud? Could you provide some more details about the problem, e.g. what you were doing when the error occurred.  

Are there any errors in the network/console tab (Fn+F12 on chrome)

**Reply by ghost** [reply]
<img width="960" alt="budi-1" src="https://user-images.githubusercontent.com/94033860/166094964-1d0c3b1c-e592-4008-99a6-76798d4728ca.png">


@melohagan  I have attached the images and, this is a self hosted budibase

**Reply by shogunpurple** [reply]
@newtonlaw95 please have a look at this https://github.com/Budibase/budibase/discussions/4471#discussioncomment-2612132 - it is likely to be the solution here. Could you try that and let me know if that works for you?

**Reply by ghost** [reply]
@shogunpurple I have tried this and added the platform url and ran `docker-compose up -d` as well still i am receiving the same error 

Sign in with google is working perfectly but when i try executing the google sheets i am getting the above error
